logger.go

```diff
@@ -21,8 +33,43 @@ func NewLogger() (*zap.Logger, error) {
 	var err error
 
 	loggerOnce.Do(func() {
-		Logger, err = zap.NewProduction()
+		lumberJackLogger := lumberjack.Logger{
+			Filename:   "logs/server.log", //Filename is the file to write logs to
+			MaxSize:    50,                //MB
+			MaxBackups: 30,                //MaxBackups is the maximum number of old log files to retain.
+			MaxAge:     90,                //days
+			Compress:   false,
+		}
+		zap.RegisterSink("lumberjack", func(*url.URL) (zap.Sink, error) {
+			return lumberjackSink{
+				Logger: &lumberJackLogger,
+			}, nil
+		})
+		zapConfig := zap.Config{
+			Level:             zap.NewAtomicLevelAt(zapcore.DebugLevel),
+			Development:       false,
```

Comment:
This value should be set according to Config.Settings.Debug (https://github.com/dreamvo/gilfoyle/blob/master/config/config.go#L21) value

Reply:
I will set the LogLevel from the config file